### PR TITLE
Add clear button to collection search input

### LINKED ISSUE #61: Add clear button to collection search input
Based on the discussion on #52 we decided to add a clear button to the collection search input.

### PR DESCRIPTION
**Pull request by dhmacs**
This PR closes #61 a fixes a bug on the collection state

## QA

1. Visit the Vercel PR preview
2. Visit any collection that contains products
3. Verify that a clear button appears when you type something on the collection search input

### CODE REVIEW

**Comment by deltuh-vee** [comment]
QA 🎬
Clear button appears when text is entered into the collection search input and clears text when pressed.

File: features/collection/components/FilterableProductSection.tsx [hunk 1]
```diff
@@ -46,6 +46,7 @@ export const FilterableProductSection = React.memo(() => {
    const productsContainerScrollRef = useScrollIntoViewEffect([hits]);
 
    if (collectionState === CollectionState.Empty) {
+      console.log('EMPTY!!');
```
Comment (cdcline) [inline features/collection/components/FilterableProductSection.tsx:49]: Do you still want this log? Seems like you don't...

Comment (dhmacs) [reply features/collection/components/FilterableProductSection.tsx:49]: @cdcline I'll remove it 😅

**Comment by masonmcelvain** [comment]
CR ✌🏻 deploy_block ✋🏻 on removing the console log that Chris mentioned

File: features/collection/components/FilterableProductSection.tsx (hunk 1 above)
Comment (dhmacs) [reply features/collection/components/FilterableProductSection.tsx]: @cdcline fixed 👍

**Comment by sterlinghirsh** [comment]
CR :zap:

**Comment by deltuh-vee** [comment]
carryover QA 🎬

**Comment by masonmcelvain** [comment]
CR ✌🏻

**Comment by masonmcelvain** [comment]
un_deploy_block ✌🏻

**Pull request merged**